tests/test_calculationBucket.py

```
@@ -0,0 +1,281 @@
+        # Should show total count and at most 5 entries
+        self.assertTrue(s.startswith("CalculationBucket (7)["))
+        self.assertIn("...", s)
+        # Test below threshold (no ellipsis)
+        bucket._current_combinations = combos[:3]
+        s2 = str(bucket)
+        self.assertFalse("..." in s2)
+
+        s3 = repr(bucket)
+        self.assertEqual(
+            s3,
+            f"CalculationBucket({DummyGeneralManager.__name__}, {"{}"}, {"{}"}, None, False)",
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: expecting '}' (tests.test_calculationBucket, line 125)'

(E0001)